src/main/java/jenkins/bouncycastle/api/InstallBouncyCastleJCAProvider.java

```diff
@@ -46,6 +47,7 @@
  */
 public class InstallBouncyCastleJCAProvider extends MasterToSlaveCallable<Boolean, Exception> {
 
+    private static Logger LOG = Logger.getLogger(InstallBouncyCastleJCAProvider.class.getName());
```

Comment:
nit: This is unused, so might as well delete it and remove the import.

Reply:
```suggestion
```